src/test/resources/json/addFileStatus_mutation_alldata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "query": "mutation addFileStatus($addFileStatusInput: AddFileStatusInput!){addFileStatus(addFileStatusInput: $addFileStatusInput){fileId,statusType,statusValue}}",
+  "variables": {
+    "addFileStatusInput": {
+      "fileId": "07a3a4bd-0281-4a6d-a4c1-8fa3239e1313",
+      "statusType": "Upload",
+      "statusValue": "Success"
+    }
+  }
+}
```

Comment:
Another line break here please. There's a few missing from these json files, if you could add them all in.

Reply:
Yeah, added in all the jsons, thanks.